src/utilities/error.h

```diff
@@ -26,16 +27,23 @@ typedef struct
 
 extern hypre_Error hypre__global_error;
 #define hypre_error_flag  hypre__global_error.error_flag
+#define hypre_error_temp_flag  hypre__global_error.temp_error_flag
 
 /*--------------------------------------------------------------------------
  * HYPRE error macros
  *--------------------------------------------------------------------------*/
 
 void hypre_error_handler(const char *filename, HYPRE_Int line, HYPRE_Int ierr, const char *msg);
+void hypre_error_handler_rank_0(HYPRE_Int myid, const char *filename, HYPRE_Int line,
```

Comment:
Remove this and the '_0' functions below.

Reply:
Done in [8ed50af](https://github.com/hypre-space/hypre/pull/1040/commits/8ed50af2b09c7125308badda38a3f57b4c1171ba)